metrics/grafana/tikv_details.dashboard.py

```diff
@@ -2547,6 +2547,28 @@ def RaftMessage() -> RowPanel:
             ),
         ]
     )
+    layout.row(
+        heatmap_panel_graph_panel_histogram_quantile_pairs(
+            heatmap_title="Raft Message Send Wait duration",
+            heatmap_description="The time consumed waiting to send Raft Message",
```

Comment:
How about:
```suggestion
            heatmap_description="The time spent on waiting to send Raft Messages",
```

Reply:
The word "consumed" is widely used in other metrics on the dashboard. I'm just keeping the style consistent. So, maybe we don't need to change it?

Reply:
I have made the change from "Message" to "Messages".